programmers/두-정수-사이의-합.js

```diff
@@ -0,0 +1,15 @@
+function solution(a, b) {
+    let sum = 0;
+    let i = 0;
+    if( b < a) {
+        for( i = b; i < a + 1; i++){
+            sum += i;
+        }
+    }
+    else {
+        for( i = a; i < b + 1; i++){
+            sum += i;
+    }
+    }
+  return sum;
+}
```

Comment:
1. 공백과 들여쓰기 신경쓰기!!

2. for문의 초기식안에서 사용될 i는 for문 안에서 선언해주기
  ex) for(초기식; 조건식; 증감식) { ... }
  ex) for(let i = 0; i < N; i++); { ... }

```suggestion
function solution(a, b) {
    let sum = 0;
    
    if (b < a) {
        for (let i = b; i < a + 1; i++){
            sum += i;
        }
    }
    else {
        for(let i = a; i < b + 1; i++) {
            sum += i;
        }
    }
  return sum;
}
```

최솟값을 구해주는 Math.min() 메서드와 최댓값을 구해주는 Math.max() 메서드를 이용해서
풀이도 가능하다.
```js
function solution(a, b) {
    const MIN_NUM = Math.min(a, b);
    const MAX_NUM = Math.max(a, b);
    let answer = 0;

    for (let i = MIN_NUM; i < MAX_NUM + 1; i++) { 
        answer += i;
    }
    
    return answer;
}
```